app/invocation/invocation_action_card.tsx

```diff
@@ -564,6 +564,19 @@ export default class InvocationActionCardComponent extends React.Component<Props
       .catch((e) => console.error(e));
   }
 
+  private getExecutionAuxiliaryMetadata(): execution_stats.ExecutionAuxiliaryMetadata | null | undefined {
```

Comment:
nit: is there some way to get this and getFirecrackerVMMetadata to share code? I'm not sure how generics work in typescript